src/_docs/form-templates/international-phone/component-example.html

```diff
@@ -13,7 +14,9 @@ <h2>Example using Form Service</h2>
         [label]="'International Phone Number'"
         [group]="form.controls.phone"
         [phoneName]="'Intl Telephone'"
-        [prefixName]="'Intl Country'"></sam-intl-telephone-group>
+        [prefixName]="'Intl Country'"
```

Comment:
@yerramshilpa Can you make the label names different on the second example? It will cause 508 defects if not. We have to have distinct ids to pass 508.

Reply:
Sure